### PR TITLE
Klaus jimenez offer refuse

### PR DESCRIPTION
**Pull request by jimenez**
Follow up for https://github.com/docker/swarm/pull/1478.

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

**Comment by vieux** [comment]
@jimenez `engine-master` tests failures are unrelated to your changes.

File: cluster/mesos/cluster.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ type Cluster struct {
 	TLSConfig           *tls.Config
 	options             *cluster.DriverOpts
 	offerTimeout        time.Duration
+	refuseTimeout       *float64
```
Comment (dongluochen) [inline cluster/mesos/cluster.go]: @jimenez Would it be better to have its type as time.Duration, to be consistent with offerTimeout and taskCreationTimeout?

Comment (jimenez) [inline cluster/mesos/cluster.go]: @dongluochen I addressed your comment PTAL

**Comment by abronan** [comment]
LGTM

**Comment by dongluochen** [comment]
LGTM.

**Pull request merged**